src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncExceptionHandlerRewriter.cs

```diff
@@ -1004,14 +1007,18 @@ private sealed class AwaitCatchFrame
             private readonly Dictionary<LocalSymbol, LocalSymbol> _hoistedLocals;
             private readonly List<LocalSymbol> _orderedHoistedLocals;
 
-            public AwaitCatchFrame(SyntheticBoundNodeFactory F, TryStatementSyntax tryStatementSyntax)
+            public AwaitCatchFrame(SyntheticBoundNodeFactory F, TryStatementSyntax tryStatementSyntax, AwaitCatchFrame parentOpt = null)
```

Comment:
>, AwaitCatchFrame parentOpt = null [](http://example.com/codeflow?start=101&length=34)

Optional parameters are error prone. It is easy to fail to pass a value when it is required. Also, attention of reviewers is not drawn to call sites that are left unmodified.